ydb/docs/en/core/reference/configuration/tls.md

```diff
@@ -1,4 +1,4 @@
-# TLS configuration
+# Section `tls`
```

Comment:
For the current content, it's not true. We'd better document those few options in this section or rename the section later.